recipes/szip/all/conanfile.py

```diff
@@ -102,17 +102,25 @@ def _create_cmake_module_alias_targets(module_file, targets):
                     set_property(TARGET {alias} PROPERTY INTERFACE_LINK_LIBRARIES {aliased})
                 endif()
             """.format(alias=alias, aliased=aliased))
-        tools.save(module_file, content)
+        save(self, module_file, content)
 
     @property
     def _module_file_rel_path(self):
-        return os.path.join("lib", "cmake", "conan-official-{}-targets.cmake".format(self.name))
+        return os.path.join("lib", "cmake", f"conan-official-{self.name}-targets.cmake")
 
     def package_info(self):
-        self.cpp_info.set_property("cmake_file_name", "szip")
+        self.cpp_info.set_property("cmake_file_name", "SZIP")
```

Comment:
```suggestion
        self.cpp_info.set_property("cmake_file_name", "szip")
```

Reply:
Why go back to lowercase?  I was aiming to follow upstream in the same way as the ZLIB library.